release-1.6/release-notes-draft.md

```diff
@@ -59,12 +59,63 @@
 
 ## Cluster Lifecycle
 ### New Features
+- kubeadm is a simple tool for creating Kubernetes clusters on existing
```

Comment:
Please, add the new features only from the [spreadsheet](https://docs.google.com/spreadsheets/d/1nspIeRVNjAQHRslHQD1-6gPv99OcYZLMezrBe3Pfhhg/edit#gid=0), that reflects the features repo.

Only the items from the "features" repo have to be present at the "New features" section; please, move other to the "notable changes" or "breaking changes".